mkdocs/docs/configuration.md

```diff
@@ -195,6 +195,19 @@ catalog:
     table-name: iceberg
 ```
 
+If you prefer to pass the credentials explicitly to the client instead of relying on environment variables,
+
+```yaml
+catalog:
+  default:
+    type: dynamodb
+    table-name: iceberg
```

Comment:
Just noticed that we're mixing case here 😭

Reply:
I think we have an inconsistency since the FileIO configurations use hyphens for s3 credentials:
https://github.com/apache/iceberg-python/blob/d8bc9a9909f77532681344762ae71713191c5cf9/mkdocs/docs/configuration.md?plain=1#L66-L67

Also, what about adding a `dynamo.` prefix to these credential keys?
```
dynamodb.access-key-id
dynamodb.secret-access-key
...
```
This could help clarify that these keys are used exclusively by the catalog, not the FileIO:

Just thinking ahead, if we're in favor of renaming these configurations, perhaps it's something we could roll out in a separate PR, especially considering updates to the GlueCatalog configurations that would follow.

I'd love to hear your thoughts on this.

Reply:
@waifairer Yes, I totally agree. Let's do that in a separate PR.